build.sbt

```
@@ -27,8 +27,8 @@ val catsV = "2.9.0"
 val catsEffectV = "3.4.9"
 val fs2V = "3.6.1"
 val http4sV = "0.23.18"
-val fiberLocalV = "0.1.1"
-val natchezV = "0.1.6"
+val fiberLocalV = "0.1.2"
+val natchezV = "0.3.1"
```

Untested, but I think I saw this exists.

```suggestion
val natchezV = "0.3.2"
```

---

Nice catch.